### DISCUSSION TITLE
The accurate ratio should be reached for a dpgen process?

### DISCUSSION THREAD

**Discussion by Guancred** [deepmodeling/dpgen#1527] [Q&A]
Hi, I have run dpgen with 55 iteration, but the accurate ratio is always remain about 98.3% for the last 13 iteration, so i have a question that whether i should run more iterations for the higher accuracy?

**Comment by wanghan-iapcm** [comment]
it does not make sense to further iterate when the acc ratio converges. 
If 98.3% is high enough depends on your requirement on the accuracy of the model .

**Reply by Niuyutao** [reply]
Hello, you have time to help me solve my problem, thank you very much, look forward to your answer! "
About the fp run in dpgen
"
